package.json

```diff
@@ -9,6 +9,7 @@
     "@ramonak/react-progress-bar": "^3.1.0",
     "@sentry/react": "^6.16.1",
     "@sentry/tracing": "^6.16.1",
+    "@strapi/provider-email-nodemailer": "^4.6.0",
```

Comment:
why is strapi package needed in frontend?

Reply:
I had previously updated the nodemailer thinking this might be the issue. So, I pushed the changes.

Reply:
it doesn't look like you updated existing nodemailer. There was no nodemailer package previously installed and you have added a new package

Reply:
@rathorevaibhav 
email-nodemailer was already installed with version 3.6 to which I updated it to 4.6

Reply:
@Rajsnehil I don't see nodemailer in the main branch package.json. Are you sure it was already installed?
[here's the link for package.json main branch](https://github.com/gitmedha/sis-frontend/blob/main/package.json)